adapters/Nend/NendAdapter/nativeads/GADMediationAdapterNendNativeForwarder.m

```diff
@@ -0,0 +1,53 @@
+//
+//  GADMediationAdapterNativeForwarder.m
+//  NendAdapter
+//
+//  Copyright © 2019 FAN Communications. All rights reserved.
+//
+
+#import "GADMediationAdapterNendNativeForwarder.h"
+#import "GADMAdapterNendConstants.h"
+#import "GADMAdapterNend.h"
+#import "GADMAdapterNendAdUnitMapper.h"
+
+@interface GADMediationAdapterNendNativeForwarder () <GADMAdapterNendNativeAdLoaderDelegate>
+
+@property(nonatomic, weak) id<GADMAdNetworkAdapter> adapter;
+@property(nonatomic, weak) id<GADMAdNetworkConnector> connector;
+
+@end
+
+@implementation GADMediationAdapterNendNativeForwarder
+
+
+- (instancetype)initWithAdapter:(id<GADMAdNetworkAdapter>)adapter
+                      connector:(id<GADMAdNetworkConnector>)connector {
+    self = [super init];
+    if (self != nil) {
+        _adapter = adapter;
+        _connector = connector;
+    }
+    return self;
+}
+
+- (void)getNativeAdWithAdTypes:(NSArray<GADAdLoaderAdType> *)adTypes options:(NSArray<GADAdLoaderOptions *> *)options {
+    id<GADMAdNetworkConnector> strongConnector = self.connector;
+
+    NSString *spotId= [GADMAdapterNendAdUnitMapper mappingAdUnitId:strongConnector paramKey:kGADMAdapterNendSpotID];
+    NSString *apiKey = [GADMAdapterNendAdUnitMapper mappingAdUnitId:strongConnector paramKey:kGADMAdapterNendApiKey];
+    GADMAdapterNendExtras *extras = [strongConnector networkExtras];
```

Comment:
This can crash if `strongConnector` is `nil`. 
Null-check `strongConnector` first.

Reply:
I'll check the object before attaching it.

Reply:
This will still crash when trying to call `mappingAdUnitId:` if `strongConnector` is `nil`.